examples/boussinesq_2d_imex/playground.py

```diff
@@ -36,12 +36,12 @@
     swparams['do_LU'] = True
 
     sparams = {}
-    sparams['maxiter'] = 12
+    sparams['maxiter'] = 16
```

Comment:
In general, yes. You might have to tweak the tolerance a bit, PFASST takes quite long to reach very fine residuals. I tried to go for something between 1e-5 and 1e-6. The residual after the first iteration is already 1e-4, so larger values do not make much sense.